setup.py

```diff
@@ -4,7 +4,7 @@
 from setuptools import setup, find_packages
 
 install_requires = [
-    'tornado<6',
+    'tornado',
```

Comment:
Was removing this necessary?
This doesn't seem to conform with the tests, which seem to require 6.0.3 .